pkg/kubelet/dockertools/manager.go

```diff
@@ -60,7 +60,7 @@ import (
 const (
 	DockerType = "docker"
 
-	MinimumDockerAPIVersion = "1.18"
+	MinimumDockerAPIVersion = "1.21"
```

Comment:
Something tells me there will be major dicussion about upping the MinimumDockerAPIVersion from 1.18 to 1.21